stl/src/sharedmutex.cpp

```diff
@@ -39,4 +40,27 @@ void __cdecl _Smtx_unlock_shared(_Smtx_t* smtx) noexcept { // unlock non-exclusi
 void __stdcall _Thrd_sleep_for(const unsigned long ms) noexcept { // suspend current thread for `ms` milliseconds
     Sleep(ms);
 }
+
+_Thrd_result __stdcall _Cnd_timedwait_for(const _Cnd_t cond, const _Mtx_t mtx, const unsigned target) noexcept {

```

Comment:
Working without a units library is scary. We should clearly name this parameter, and the `GetTickCount64()` variables, to mark that they're milliseconds.